extensions/wrapper/clients/typescript-client/src/oauth2/FetchUtils.ts

```diff
@@ -0,0 +1,34 @@
+import {z} from 'zod';
+
+const ClientCredentialsResponseSchema = z.object({
+    access_token: z.string().min(1),
+    token_type: z.string(),
+    expires_in: z.number(),
+    scope: z.string(),
+});
```

Comment:
i hope there are no backdoors that this reponse might contain further stuff. i hope zod does not overreact, if there's other stuff in there.

Reply:
it doesnt I checked with my test oauth backend